CefSharp.Wpf/Internals/MouseTeleport.cs

```diff
@@ -0,0 +1,55 @@
+using CefSharp.Structs;
+
+namespace CefSharp.Wpf.Internals
+{
+    public class MouseTeleport
```

Comment:
Terrible class name IMHO, doesn't tell you anything about it's purpose for being.

It translates/adjusts mouse coordinates when popup is visible.

Reply:
Thanks for your advice, I appreciate that and agree. I renamed it to "MouseAdjustor". Is that okay?

Reply:
Hmm, go with that for now. See if we can come up with something better later.